animations/cascaded-animation.html

```diff
@@ -79,11 +60,15 @@
         config.timing.delay += nodeDelay;
         config.node = node;
 
-        var animation = new animationConstructor();
-        var effect = animation.configure(config);
+        var animation = document.createElement(config.animation);
```

Comment:
Previously, animations would be:

 - created once for every registered animation
 - configured once for every practical usage of the animation

Now we *create it* once for *every* practical usage. This seems like a performance regression.